searchlib/src/vespa/searchlib/queryeval/intermediate_blueprints.cpp

```diff
@@ -243,6 +243,11 @@ AndBlueprint::createIntermediateSearch(const MultiSearch::Children &subSearches,
     return SearchIterator::UP(search);
 }
 
+ExecuteInfo
+AndBlueprint::computeNext(const Blueprint & child, const ExecuteInfo &execInfo) const {
+    return ExecuteInfo(execInfo.isStrict(), (execInfo.hitRate()*child.getState().estimate().estHits)/get_docid_limit());
```

Comment:
the blueprint already has a function called hit_ratio that does this conversion (estimate to ratio)

Reply:
Done, using that.